src/braket/task_result/quera_metadata_v1.py

```diff
@@ -0,0 +1,39 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License
+
+from pydantic import Field, conint
+
+from braket.schema_common import BraketSchemaBase, BraketSchemaHeader
+
+
+class QueraMetadata(BraketSchemaBase):
+    """
+    The Querametadata result schema.
+
+    Attributes:
+        braketSchemaHeader (BraketSchemaHeader): Schema header.
+            Users do not need to set this value. Only default is allowed.
+        executedProgram (str): The program executed on the QPU.
+
+    Examples:
+        >>> QueraMetadata(executedProgram="DECLARE ro BIT[2];")
```

Comment:
This is a wrong example. Please include a valid QuEra program example.

Reply:
This is a incorrect notes. For some of gate based program. We have compiled program as additional meta data in result. We don't do this in quera. So numSuccessfulShots should be right one here. Will change now